website/source/docs/synced-folders/nfs.html.md

```diff
@@ -134,10 +134,9 @@ If you do not want to type your password on every `vagrant up`, Vagrant
 uses thoughtfully crafted commands to make fine-grained sudoers modifications
 possible to avoid entering your password.
 
-Below, we have a couple example sudoers entries. Note that you may
-have to modify them _slightly_ on certain hosts because the way Vagrant
-modifies `/etc/exports` changes a bit from OS to OS. If the commands below
-are located in non-standard paths, modify them as appropriate.
+Below are example entries to add at the end of your sudoers file.
+
+Note that you may have to modify them _slightly_ on certain hosts because the way Vagrant modifies `/etc/exports` changes a bit from OS to OS. If the commands below are located in non-standard paths, modify them as appropriate.
```

Comment:
I think instead of splitting up this paragraph, we should simply add an additional one below it talking about how the suoders file format applies these rules in order of entry. Meaning that the order in which you add these rules to this file matters, so pay attention. We could even quote the sudoers man page here:

>   When multiple entries match for a user, they are applied in order.  Where there are multiple matches, the last
     match is used (which is not necessarily the most specific match).

And tell users to refer to the man page for how to use this file.

Reply:
Thanks Brian! I put everything in a single paragraph where you recommended.